Added Book template

WinnerWind opened the pull request:
Added Book template
# Book
## A simple minimalist plug and play book template for Obsidian. Works out of the box without any YAML

WinnerWind committed:
Added Book template

Added Book template

universvm commented:
Hi @WinnerWind ,

Thanks for your contribution. I'd be very happy to include it, however could you do some modifications? 

Have a look at: https://github.com/universvm/obsidian-pandoc-templates/tree/main/dissertation for a template on the README. Try to follow the way it is laid out (with the same titles). I'd like to keep templates more concise with about 1 sentence description.

Also could you add support for citations?  It would simplify the workflow for people wanting to use your template in a professional environment. Have a look at how natbib is imported here: https://github.com/universvm/obsidian-pandoc-templates/blob/main/dissertation/dissertation.tex

Btw, the addition of citation support does not affect your ability to use the template without YAML input.

WinnerWind commented:
Hi there!
I felt that the Readme of the dissertation was too short, so I made my own, but at your request, I'll make it a simple one sentence.
Though, this is a template made to emulate real books (which often don't have citations) I'll try to add citations. I'll let you know if I succeed. Do give me some time.
Thanks!

universvm commented:
Take your time and thank you!

universvm commented:
Hi there,

Perhaps it'd help if you setup a bib file and cite in the file. If integrated correctly there should be citations at the end of the document. 

This might be a bit long but you can follow the tutorial here: https://betterhumans.pub/obsidian-tutorial-for-academic-writing-87b038060522

WinnerWind commented:
Thank you so much! I'm looking into it now.

WinnerWind committed:
Changed README

WinnerWind commented:
Hi there!
Just an update : I wasn't able to figure out how to do the citations. I'm still working on it.

WinnerWind committed:
Fixed Example and added alternate template

I have fixed the example image and the example PDF. Their headings were broken as they lacked one space.
I have also added the alternate template book_with_citations.tex. This should hopefully ship with citation support.

WinnerWind commented:
Hi there! 
I think that I have got it to work. I have included both the book without citations template, and the book with citations template. Please do check them out and let me know if they work for you.
Book with citations may still be broken, as honestly I am not sure how to get it to work. It seems to export fine on my setup though.
Thanks!

universvm commented:
Hi @WinnerWind thanks for all the work, it looks great! 

One small thing I'd suggest is to use one template only and add an if condition. if the user passes a bib file, then do citations. ChatGPT should be able to help you out with this. Also it seems like you include the bib file in the template but the example png/pdf does not contain any citations. Would it be possible to only include an example with citations? 

Thanks in advance for this!

universvm commented:
Hi @WinnerWind , no worries at all, take your time and thanks again for the contribution!

WinnerWind committed:
Add files via upload

I gave up trying to fix citations.

WinnerWind commented:
Hi there!
I am so sorry but after two weeks of trying and a lot of headaches later, I was not able to add citations. 
I am really sorry that this is the case. I don't know how to add it, even though I have analyzed every inch of `dissertation.tex`. I'm still pretty new to LATEX.
I hope that one day I can add it, but I have to throw in the towel for now.

universvm commented:
@WinnerWind  don't worry about it, I will continue your work whenever I have more time. Thanks for everything you've done so far!

WinnerWind committed:
Update book.tex

WinnerWind committed:
Update book.md

Change example text to something I like.

WinnerWind committed:
Change Book.PDF and Image into something I like.

WinnerWind committed:
delete Redundant Book.bib

WinnerWind commented:
Hi there! I'm going to give this one another shot. No gaurentees though.

universvm commented:
Thanks @WinnerWind !
